extensions/guacamole-auth-totp/src/licenses/LICENSE

```diff
@@ -284,6 +284,15 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
+JAXB API / XML Processing (https://projects.eclipse.org/projects/ee4j.jaxb)
```

Comment:
Sorry - I was wrong here. There are a number of other licenses cited in `LICENSE` for TOTP, and the "JAXB ..." one should thus go even further up where it's properly alphabetical.

Reply:
Now I've said my A B Cs, next time won't you sing with me... :musical_note: